### PR TITLE
Lms

### PR DESCRIPTION
**Pull request by muhammadqamar**

### CODE REVIEW

**Review by i-do-dev — APPROVED** [review]

File: src/store/actions/project.js [hunk 1]
```diff
@@ -336,127 +327,147 @@ export const getProjectCourseFromLMS = (
...
+    allowOutsideClick: false,
+  });
+  const response = await projectService.fetchLmsDetails(lms, projectId, settingId);
+
+  if (response.project) {
+    const globalstoreClone = store.getState();
+
+    dispatch(setLmsCourse(response.project, globalstoreClone));
+    const counterarray = [];
+    const { token } = JSON.parse(localStorage.getItem('auth'));
```
Comment (simonebelotti75) [inline src/store/actions/project.js:421]: Is this correct?

File: src/store/actions/project.js [hunk 2]
```diff
@@ -336,127 +327,147 @@ export const getProjectCourseFromLMS = (
...
+          showConfirmButton: false,
+          allowOutsideClick: false,
+        });
+        const globalstoreClone = store.getState();
+        const playlistcounter = !!globalstoreClone.project.lmsCourse
+                && globalstoreClone.project.lmsCourse.playlists_copy_counter
+                  .length > 0
+          ? globalstoreClone.project.lmsCourse.playlists_copy_counter
+          : 0;
+        console.log(playlistcounter);
```
Comment (simonebelotti75) [inline src/store/actions/project.js:446]: Remove console log

File: src/store/actions/resource.js [hunk 3]
```diff
@@ -449,7 +449,10 @@ export const removeShareActivity = async (activityId, resourceName) => {
   }
 };
 
-export const loadH5pShareResource = (activityId) => resourceService.loadH5pShared(activityId);
+export const loadH5pShareResource = async (activityId) => {
+  const result = await resourceService.loadH5pShared(activityId);
+  return result;
+};
```
Comment (simonebelotti75) [inline src/store/actions/resource.js:455]: Do we need this change?
I already saw this several times but I don't think it is necessary

File: src/store/actions/search.js [hunk 4]
```diff
@@ -17,15 +17,16 @@ export const cloneProject = (projectID) => {
   searchService.cloneProject(projectID);
 };
 
-export const clonePlaylist = (projectId, playlistId) => {
-  searchService.clonePlaylist(projectId, playlistId);
+export const clonePlaylist = (projectId, playlistid) => {
+  searchService.clonePlaylist(projectId, playlistid);
```
Comment (simonebelotti75) [inline src/store/actions/search.js:21]: Revert this change

File: src/store/actions/search.js [hunk 5]
```diff
@@ -17,15 +17,16 @@ export const cloneProject = (projectID) => {
...
 
-export const cloneActivity = (playlistId, activityId) => {
-  searchService.cloneActivity(playlistId, activityId);
+export const cloneActivity = (playlistid, activityId) => {
+  searchService.cloneActivity(playlistid, activityId);
 };
 
-// export const advancedSearches = (searchQuery) => {
+// export const advancedSearches = (searchquerry) => {
+//   console.log(searchquerry);
```
Comment (simonebelotti75) [inline src/store/actions/search.js:29]: Revert this change

File: src/store/actions/search.js [hunk 6]
```diff
@@ -17,15 +17,16 @@ export const cloneProject = (projectID) => {
...
-export const clonePlaylist = (projectId, playlistId) => {
-  searchService.clonePlaylist(projectId, playlistId);
+export const clonePlaylist = (projectId, playlistid) => {
+  searchService.clonePlaylist(projectId, playlistid);
 };
 
-export const cloneActivity = (playlistId, activityId) => {
-  searchService.cloneActivity(playlistId, activityId);
+export const cloneActivity = (playlistid, activityId) => {
+  searchService.cloneActivity(playlistid, activityId);
```
Comment (simonebelotti75) [inline src/store/actions/search.js:25]: Revert this change

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]

**Review by simonebelotti75 — APPROVED** [review]
